department-of-veterans-affairs/va.gov-cms
# VACMS-9097 facility title change notification

## Description
Relates to #9097. Has the code for sending notifications to the help desk based on facility title and vet center official name updates.

## Testing done
Changed a facility title, verified emails sent
Changed a vet center, verified different emails sent

## Screenshots
<img width="1594" alt="Screen Shot 2022-06-13 at 3 36 22 PM" src="https://user-images.githubusercontent.com/11279744/173458923-e65efed3-2c20-49db-bed7-aa4074622ee9.png">
<img width="1594" alt="Screen Shot 2022-06-13 at 3 36 32 PM" src="https://user-images.githubusercontent.com/11279744/173458928-2d6c2a81-ad35-4dc8-938d-550f7432a1a0.png">


## QA steps

As an admin user: 

- Change the name of a Facility (VBA/NCA/VAMC). 
  - Depending on the facility chosen, you may need to manually delete the `disabled` tag from the form input element. 

- Change the name of a Vet Center. 
  - You may need to create a new vet center and then change the official name... there is some validation going on around changing this field (for good reason, but makes it hard to test this on tugboat)

- See that emails were generated from two separate templates for the facility types. See that only one email was fired per node title changed.

### Select Team for PR review
